ebot7/sample-applications
# Feature/ap 356 sample event handlers

### Changes
Adds a sample event handler that parses an event from AP, uses the SDK to fetch the messages from the conv, and prints them all in CloudWatch.

### To Test
- Run the following Curl request
```bash
curl --location --request POST 'https://qidsdzl6c8.execute-api.eu-central-1.amazonaws.com/prod/transcript' \
--header 'Content-Type: application/json' \
--data-raw '{
  "eventName": "message:created",
  "timestamp": 1623760938357,
  "data": {
    "id": "60c8a02aa1d42926dcbbf9ca",
    "promptOptions": [],
    "botId": "60be0c3d04d81049a85d409b",
    "convId": "60d2ddad0a87f8ab07a08109",
    "orgId": "605200a9249e541159c97c09",
    "body": "Direct HTTP call",
    "source": "visitor",
    "botStatus": "60785e68ab7c2eb31f19a208",
    "isGreeting": false,
    "seqInConv": 2,
    "createdAt": "2021-06-15T12:42:18.357Z",
    "updatedAt": "2021-06-15T12:42:18.357Z",
    "apCreateReceivedAt": null,
    "apUpdateReceivedAt": null
  }
}'
```
- Confirm that the response is`successfully created transcript`
- Confirm that there are logs in CloudWatch (requires access to the ebot7-integrations AWS org.
